src/crypto/ProtectionFormat.ts

```diff
@@ -0,0 +1,15 @@
+
+ /**
+  * Enum to define different protection formats
+  */
+export enum ProtectionFormat {
+  /**
```

Comment:
yes because it is used in the IKeyStore interface

Reply:
It looks like `ProtectionFormat` is also [defined here](https://github.com/decentralized-identity/did-auth-jose/blob/master/lib/keyStore/ProtectionFormat.ts), I'm still not 100% clear where the version in this PR is being used, since it doesn't appear to be imported in any other file here

Reply:
This package reuses the keystore classes from did-auth-jose. It also exports them because a wallet will use hub sdk but probably not did-auth-jose directly